lib/clock.js

```diff
@@ -224,6 +223,7 @@ async function clockBenchmark(bench, recommendedCount) {
   result[0] = Math.max(MIN_RESOLUTION, result[0]);
   for (const p of bench.plugins) {
     if (typeof p.onCompleteBenchmark === 'function') {
+      // TODO: this won't work when useWorkers=true
```

Comment:
We can support this by using the same strategy implemented on pino, which allows them to pass the path of the plugin (js file) and they load the js file in the worker, is not something "pretty" but works well.

Reply:
I think it's easier just to use the same approach I did for `Benchmark.serializeBenchmark`

Reply:
The only issue with that approach is your benchmark needs to be isolated, you won't be able to import/reuse external code other than the code that is inside the benchmark.

But with unmanaged timer, we can solve the issue of the cost of importing external code, so it could work well.

Reply:
That's correct. There are other nuances to solve as some states will be created before the worker starts and the rest when the worker is instantiated - so they need to share a similar state. I will leave it as a TODO for now.